ios/keyboardObserver/REAKeyboardEventObserver.m

```diff
@@ -193,6 +206,15 @@ - (void)recognizeInitialKeyboardState
   });
 }
 
+- (void)clearListeners
+{
+  RCTUnsafeExecuteOnMainQueueSync(^() {
+    [self->_listeners removeAllObjects];
+    [self->displayLink invalidate];
+    self->displayLink = nil;
+  });
+}
+
```

Comment:
Shouldn't this also remove the `self` observer on the notification center, like the `unsubscribeFromKeyboardEvents:listenerId` method does?